src/Fmt_ast.ml

```diff
@@ -2016,6 +2016,8 @@ and fmt_structure_item c ~sep ~last:last_item {ctx; ast= si} =
   wrap_k fmt_cmts_before fmt_cmts_after
   @@
   match si.pstr_desc with
+  | Pstr_attribute ({txt= "ocamlformat.disable"; _}, _) ->
+      raise Formatting_disabled
```

Comment:
An analogous case for signatures is also needed I think.

Reply:
👍